mailet/icalendar/src/test/java/org/apache/james/transport/mailets/ICALToJsonAttributeTest.java

```diff
@@ -311,6 +351,7 @@ public void serviceShouldAttachJsonForSeveralRecipient() throws Exception {
                         "\"ical\": \"" + toJsonValue(ics) + "\"," +
                         "\"sender\": \"" + SENDER.asString() + "\"," +
                         "\"recipient\": \"" + MailAddressFixture.ANY_AT_JAMES2.asString() + "\"," +
+                        "\"replyTo\": \"" + SENDER.asString() + "\"," +
```

Comment:
as said upper, I'm not sure it makes sense to add replyTo when not initially specified